sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -70,28 +66,6 @@ public DigitalTwinsClient(Uri endpoint, TokenCredential credential)
         /// enabling <see href="https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Diagnostics.md">diagnostics</see>,
         /// and controlling <see href="https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Configuration.md">retry strategy</see>.
         /// </remarks>
```

Comment:
remarks are still there but no example as we don't have the interactive login snippet anymore.